autopts/bot/iut_config/zephyr.py

```diff
@@ -120,6 +120,16 @@
         ]
     },
 
+    "csip_privacy.conf": {
+        "overlay": {
+            'CONFIG_BT_PRIVACY': 'y',
+            'CONFIG_BT_RPA_TIMEOUT': '120',
```

Comment:
OK (was asking since apparently PTS allows to tune this value via ixit)

Reply:
This will not work.  CSIP requires CSIS, which means the overlay-le-audio.conf plus the enabling of privacy.  Does this mean I have to repeat the entire overlay-le-audio.conf in "csip_privacy.conf"?
This opens a more principal discussion.  We are qualifying a product.  The product configuration has PRIVACY enabled.  This requires us to run all the Qualification tests with that configuration, we cannot run some tests with one product configuration and other tests with a different product configuration.  If we do that, we are not qualifying "A PRODUCT".

Reply:
from what I see in test spec RPA timeout is configured via ixit so this doesn't have to be 120seconds,  it can be 30 (ie just like in privacy.conf) so this should do

"pre_overlay": ["overlay-le-audio.conf, privacy.conf"]

Reply:
Yes - sure, can dio that, but then I will need to change the PIXIT configuration in csip.py as well.
Done that.